airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -282,6 +282,9 @@ def __init__(
         security_context: dict | None = None,
         container_security_context: dict | None = None,
         dnspolicy: str | None = None,
+        dns_config: k8s.V1PodDNSConfig | None = None,
+        hostname: str | None = None,
+        subdomain: str | None = None,
```

Comment:
Can you please add these parameters to the docstring above?

Reply:
Good catch, I had forgotten to add them. I have added them now.